modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -67,8 +67,9 @@ During the live migration, both OVN-Kubernetes and OpenShift SDN run in parallel
 ** `InternalTransitSwitchSubnet`
 ** `internalJoinSubnet`
 
-* Unless otherwise configured, OVN-Kubernetes uses the following IP address ranges:
-** `100.64.0.0/1`. This IP address range is used for the `internalJoinSubnet` parameter of OVN-Kubernetes by default. If this IP address range is already in use, enter the following command to update it to `100.63.0.0/16`:
+* Unless otherwise configured, OVN-Kubernetes uses the following IP address ranges: `100.64.0.0/16`, `169.254.169.0/29`, `100.88.0.0/16`, `fd98::/64`, `fd69::/125`, and the `fd97::/64`. If your cluster uses OVN-Kubernetes, do not include any of these IP address ranges in any other CIDR definitions in your cluster or infrastructure.
```

Comment:
```suggestion
* Unless otherwise configured, OVN-Kubernetes uses the following IP address ranges: `100.64.0.0/16`, `169.254.169.0/29`, `100.88.0.0/16`, `fd98::/64`, `fd69::/125`, and `fd97::/64`. If your cluster uses the OVN-Kubernetes networking plugin, do not include any of these IP address ranges in any other CIDR definitions in your cluster or infrastructure.
```
Does this mean that these are default values and they should not be explicitly configured or you get service interruption during migration/break something?

Reply:
These are default values for ovn-kubernetes,  if these values are used in the infrastructure network or as cluster/service CIDRs with openshift-sdn, it will break the migration from openshift-sdn to ovn-kubernetes.